fabric_plugin/tasks.py

```diff
@@ -273,6 +292,10 @@ def upload_script(self, script_path):
             remote_path_suffix)
         self._sftp.put(script_path, self.remote_script_path)
 
+    @staticmethod
+    def is_py_script(script_path):
+        return True if script_path.endswith('.py') else False
```

Comment:
this is just `return script_path.endswith('.py')`

Reply:
Done